projects/optic/src/commands/diff/diff.ts

```diff
@@ -78,47 +78,48 @@ export const registerDiff = (cli: Command, config: OpticCliConfig) => {
     })
     .addHelpText('after', helpText)
     .description(description)
-    .argument('[file_to_compare_against]', 'path to file to compare with')
-    .argument('[file_path]', 'path to file to compare')
-    .option(
-      '--base <base>',
-      'the base ref to compare against. Defaults to HEAD. Also supports optic cloud tags (cloud:tag_name)',
-      'HEAD'
+    .argument('[file_to_compare_against]', 'Path to file to compare with')
+    .argument('[file_path]', 'Path to file to compare')
```

Comment:
should we rename these to "before" and "after"? feels like that's what people are trying to get to anyway as they parse this.

Reply:
i can't decide how i feel about it, 

```
➜ npx ts-node $HOME/code/optic/projects/optic/src/index.ts diff --help | grep -A2 Arguments
Arguments:
  before                         Path to file to compare with
  after                          Path to file to compare

➜ optic diff --help | grep -A2 Arguments
Arguments:
  file_to_compare_against    path to file to compare with
  file_path                  path to file to compare
```

yeah, i cant decide what i like. `diff --help` uses "file1" and "file2", which is about as distilled and simple as you could make it.

Reply:
ok @acunniffe i tried the file1/file2 thing. what do you think?

```
➜ npx ts-node $HOME/code/optic/projects/optic/src/index.ts diff openapi.yml diff --help
Usage:
  optic diff <file> --base <base>
  optic diff [file1] <file2>
  optic diff [file1] <file2> --check

Run a diff between two API specs

Arguments:
  file1                          Path to file to compare with
  file2                          Path to file to compare
```

Reply:
went with file1/2